src/utils/convert.js

```diff
@@ -115,11 +116,20 @@ const renderField = (definition, key) => {
     props.push(renderProp('search', quote(definition.searchfill)));
   }
 
-  props = props.map(s => ' '.repeat(8) + s);
+  props = props.map(s => ' '.repeat(indent + 2) + s);
+  const padding = ' '.repeat(indent);
 
-  return `      {
+  return `${padding}{
 ${props.join(',\n')}
-      }`;
+${padding}}`;
+};
+
+const renderFields = (fields, indent = 0) => {
+  const result = [];
```

Comment:
I'd prefer to make this plural `results` just to more clearly indicate it's a list of things, not a single thing/object.

Reply:
Fixed in 990f1c9.